pageserver/src/tenant.rs

```diff
@@ -436,6 +435,30 @@ pub enum GetTimelineError {
     },
 }
 
+#[derive(Debug, thiserror::Error)]
+pub enum LoadLocalTimelineError {
+    #[error("FailedToLoad: {source}")]
+    FailedToLoad {
+        #[source]
+        source: anyhow::Error,
+    },
+    #[error("FailedToResumeDeletion: {source}")]
+    FailedToResumeDeletion {
+        #[source]
+        source: anyhow::Error,
+    },
+}
```

Comment:
Was thinking does it make sense to have `{source}` in the display and `#[source]`, because if we would only use `anyhow::Error::new(e: LoadLocalTimelineError)` for reporting, it would report the cause chain, see https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=ed4b2ea17ea8b72e0e7428476e11c033

Reply:
Maybe this type was never meant to be an error, if so it could be a plain enum without thiserror?